File: API/TestAPI/Entities/Subarea.cs
using System.ComponentModel.DataAnnotations;

namespace TestAPI.Entities
{
    public class Subarea
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int AreaId { get; set; }
        public Area Area { get; set; }
        public int DepartmentId { get; set; }
        public Department Department { get; set; }
    }
}

File: UI/TestUI/src/app/models/subarea.ts
export interface subarea {
    id: number
    name: string
    areaId: number
    areaName: string
    departmentId: number
    departmentName: string
    image: string
}

export interface menuSubarea {
    id: number
    name: string
}

export interface subareaCreationDTO {
    name: string
    image: File
    areaId: number
    departmentId: number
}